R/font-bitstream-vera.R

```diff
@@ -34,21 +35,21 @@ font_bitstream_vera <- function(variant = "sans", style = "roman",
   ))
 }
 
-font_bitstream_vera_files <- list(
-  sans = c(
-    roman = "Vera",
-    bold = "VeraBd",
-    oblique = "VeraIt",
-    `bold-oblique` = "VeraBI"
+font_bitstream_vera_files <- font_files(font_bitstream_vera,
+  sans = list(
+    roman = font_sans("Vera"),
```

Comment:
hmm when I look in `?par` they mention "sans", "serif" and "mono" for `family` and "plain", "bold", "italic" and "bold italic" for `font` (which in fontquiver is called `face`).